api/routes/device.go

```diff
@@ -222,23 +222,19 @@ func (h *Handler) DeleteTag(c apicontext.Context) error {
 
 func (h *Handler) RenameTag(c apicontext.Context) error {
 	var req struct {
-		NewName string `json:"new_name"`
+		Name string `json:"name"`
 	}
 
 	if err := c.Bind(&req); err != nil {
 		return err
 	}
 
-	if err := h.service.RenameTag(c.Ctx(), models.UID(c.Param("uid")), c.Param("name"), req.NewName); err != nil {
+	if err := h.service.RenameTag(c.Ctx(), c.Param("name"), req.Name); err != nil {
 		switch err {
 		case services.ErrUnauthorized:
 			return c.NoContent(http.StatusForbidden)
 		case services.ErrInvalidFormat:
 			return c.NoContent(http.StatusBadRequest)
-		case services.ErrMaxTagReached:
-			return c.NoContent(http.StatusForbidden)
-		case services.ErrNotFound:
-			return c.NoContent(http.StatusNotFound)
```

Comment:
We are running asynchronously so we can have an `ErrNotFound` if someone else renamed at the same time. We don´t know which one wins the race but one of them will receive a not found.